app/hotels/src/navigation/NewNavigationStack.js

```diff
@@ -1,15 +1,24 @@
-// @flow
+// @flow strict
```

Comment:
Shall we use `flow strict` everywhere now?

Reply:
I have not enforced it strictly. 

The difference is you can not use `Function`, `any` `Object` etc.

I use it where it makes sense. If I need to use `any` or `Object` for some reason, I use just `Flow`